obspy/io/dmx/__init__.py

```diff
@@ -0,0 +1,47 @@
+"""
+obspy.io.dmc - INGV DMX file format reader for ObsPy
+===========================================================
+
+Functions to read waveform data from the standard INGV DMX format.
+
+:author:
+    Thomas Lecocq
+    Andrea Cannatta
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
+
+
+Reading the waveforms
+---------------------
+Reading DMX is handled by using ObsPy's standard
+:func:`~obspy.core.stream.read` function. The format can be detected
+automatically, however setting the format parameter as "DMX" lead to a
+speed up.
+One optional keyword argument is available: ``station``. It is automatically
+passed to the :func:`obspy.io.dmx.core._read_dmx`. Its format 
+
+>>> import obspy
+>>> from obspy.core.util import get_example_file
+>>> filename = get_example_file('181223 120000.DMX')
+>>> # these two are equivalent, but the second case should be faster:
+>>> st = obspy.read(filename)
+>>> st = obspy.read(filename, format='DMX')
```

Comment:
These will be executed as doctests, so might as well add a 

```
>>> print(st)  # doctest: +ELLIPSIS
<trace info here>
```

Also, I think we usually use the following as example code
```
>>> from obspy import read
>>> read(...)
```

Also, you can use `read('/path/to/181223 120000.DMX')` instead of the util helper function in the example.

Also also also.. can we please get rid of the space in the test filename, pleeeeaaaaaase :)

Reply:
I copied the example doc from rg16 :-) so it might be good to do it there too at some point :)

OK for the space (but it's like that ... you know...)

Reply:
done